src/test/java/net/sf/jabref/logic/layout/format/WrapFileLinksTest.java

```diff
@@ -0,0 +1,92 @@
+package net.sf.jabref.logic.layout.format;
+
+import java.util.Collections;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+
+public class WrapFileLinksTest {
+
+    private WrapFileLinks formatter;
```

Comment:
It is a quite flexible formatter for the file field. The things I have not tested relates to the actual search path.

See http://help.jabref.org/en/CustomExports

Regarding renaming, it is always tricky for layout formatters as people may have used them. On the other hand that is disconnected now, but still a point in consistency. Name is not really clear though.